frontend/frontend/src/components/GameSummary.vue

```diff
@@ -62,7 +73,16 @@ export default {
 	},
 	methods: {
 		routeToHome: function () {
-			this.$router.push("/home");
+			this.$router.push('/home');
+			//console.log("hey");
+		},
+		artistsList: function () {
```

Comment:
cut this function

Reply:
do we not want to allow users to play a random game without selecting a specific artist?

Reply:
what function? removed the commented code, but why remove either returnToHome or artistsList function?